test/functional/apps/home/_sample_data.ts

```diff
@@ -125,8 +125,9 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         await pieChart.expectPieSliceCount(4);
         log.debug('Checking area, bar and heatmap charts rendered');
         await dashboardExpect.seriesElementCount(15);
+        // The saved search of data explorer now renders 100 lines max
         log.debug('Checking saved searches rendered');
-        await dashboardExpect.savedSearchRowCount(50);
+        await dashboardExpect.savedSearchRowCount(100);
```

Comment:
is this a bug?

Reply:
i have documented some issues with save search rendering on dashboard: https://github.com/opensearch-project/OpenSearch-Dashboards/issues/5117

Look like previously the old discover use a fixed window size with a scrolling view, the max window size is about 8 lines, and one page is 50 lines max. 

@ananzh

Reply:
original discover has `infinite` rows when scrolling down. data grid using pagination. currently we set default pagination to 100. I think it should not be counted as a bug.